### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by Nenad85**
remove `wiatForAngular()` function form tests and small fixes

### CODE REVIEW

**Review by stefannikolic — APPROVED** [review]
Everything is okay.

File: cypress/integration/dib-travel/company-settings/approval-process.spec.ts [hunk 1]
```diff
@@ -120,7 +122,8 @@ describe('Company Settings - Approval Process', () => {
     deleteApprovalProcess(approvalProcessGroup.traveler.firstName);
   });
 
-  it('should check if selected traveler group already has approval process', () => {
+  //TODO: This should be revisit after solving a problem with stripe.
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/approval-process.spec.ts]: Please follow right pattern for naming TODO

File: cypress/integration/dib-travel/company-settings/approval-process.spec.ts [hunk 2]
```diff
@@ -120,7 +122,8 @@ describe('Company Settings - Approval Process', () => {
...
+  xit('should check if selected traveler group already has approval process', () => {
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/approval-process.spec.ts]: We shouldn't use `xit`. Please comment out the whole block instead.

File: cypress/integration/dib-travel/company-settings/approval-process.spec.ts [hunk 3]
```diff
@@ -130,12 +133,12 @@ describe('Company Settings - Approval Process', () => {
...
 
     cy.get('.cdk-overlay-container simple-snack-bar > span').should(
-      'contain',
+      'have.text',
       'Selected traveler group already has approval process!'
     );
   });
 
-  it('should delete approval process for traveler group (this overrides any travel policy)', () => {
+  xit('should delete approval process for traveler group (this overrides any travel policy)', () => {
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/approval-process.spec.ts]: Same as above

File: cypress/integration/dib-travel/company-settings/subscription.spec.ts [hunk 4]
```diff
@@ -48,22 +48,22 @@ describe('Company Settings - Subscription', () => {
     addGroup(group.name, group.description, false);
   });
 
-  after(() => {
+  // TODO: This should be revisit after solving a problem with stripe.
+  /*after(() => {
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription.spec.ts]: Please revert this. We should keep this logic separated from the test itself.

File: cypress/integration/dib-travel/company-settings/subscription.spec.ts [hunk 5]
```diff
@@ -225,7 +225,8 @@ describe('Company Settings - Subscription', () => {
     cy.get('.cdk-overlay-container confirmation-dialog').should('not.exist');
   });
 
-  it('should buy new license for subscription', () => {
+  //TODO: This should be revisit after solving a problems with stripe.
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription.spec.ts]: Same as above.

File: cypress/integration/dib-travel/company-settings/subscription.spec.ts [hunk 6]
```diff
@@ -225,7 +225,8 @@ describe('Company Settings - Subscription', () => {
...
+  xit('should buy new license for subscription', () => {
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription.spec.ts]: Same as above.

File: cypress/integration/dib-travel/company-settings/subscription.spec.ts [hunk 7]
```diff
@@ -383,5 +385,9 @@ describe('Company Settings - Subscription', () => {
       'not.contain',
       ' Nov 18, 2021 '
     );
+
+    cy.visit('/people-management/groups');
```
Comment (sjova) [inline cypress/integration/dib-travel/company-settings/subscription.spec.ts]: See the above comment related to this. This logic should stay separated

**Review by sjova — CHANGES_REQUESTED** [review]
We should resolve all suggestions before the merge.

**Review by sjova — APPROVED** [review]

**Pull request merged**